android/src/main/java/com/thebluealliance/androidclient/adapters/MatchListAdapter.java

```diff
@@ -34,6 +34,11 @@ public void setTeamKey(String teamKey) {
 
     @Override
     public View getChildView(int groupPosition, int childPosition, boolean isLastChild, View convertView, ViewGroup parent) {
+        if (groupPosition >= mGroups.size()
```

Comment:
I tried to find the root cause and couldn't get anywhere. You want to give it a shot?

I figure we'll come across it in the future, it'll still be obvious when it happens, just won't crash